Remove mention of `Multi-Org Usage` page (#13047)
DataDog/documentation: content/en/account_management/multi_organization.md

## Multi-org usage

> The parent-organization can view the total and billable usage of all their organizations (child and parent organizations) by hovering over their username at the bottom left and then navigating to: `Plan & Usage`--> `Usage`.

Reviewer: ```suggestion
The parent-organization can view the total and billable usage of all their organizations (child and parent organizations) by hovering over their username on the bottom left corner and navigating to **Plan & Usage** > **Usage**.
```

### Individual organization usage

> On the Individual Organizations usage tab, you are able to view the usage of your child organizations in either absolute units or as a percent of total usage.

Reviewer: ```suggestion
On the **Individual Organizations** usage tab, you can view the usage of your child organizations in absolute units or as a percentage of total usage.
```
